client-app/core/utilities/date/index.ts

```diff
@@ -0,0 +1,22 @@
+export function addTimeToStartDate(dateOnly?: string): string | undefined {
+  if (dateOnly) {
+    return toDateObject(dateOnly).toISOString();
+  }
+}
+
+export function addTimeToEndDate(dateOnly?: string): string | undefined {
```

Comment:
addDays(dateOnly: string, days: number)
or
addOneDay(dateOnly)

Reply:
In both these functions we add "TXX:XX:XX.XXZ" to the date, that's why they named this way

Reply:
Renamed